cmp/grid/GridModel.js

```diff
@@ -303,6 +309,7 @@ export class GridModel extends HoistModel {
         autosizeOptions = {},
         restoreDefaultsWarning = GridModel.DEFAULT_RESTORE_DEFAULTS_WARNING,
         fullRowEditing = false,
+        highlightRowOnClick = XH.isPhone || XH.isTablet,
```

Comment:
I'd suggest we default to `XH.isMobileApp` (and adjust the doc comment accordingly).

Reply:
Curious about this suggestion - I think this is valuable due to the touchscreen common to both phones and tablets. Defaulting true for one touchscreen device but not the other seems a bit arbitrary.

Reply:
I was thinking that since we most commonly expect tablets to run the desktop app, they will typically have master/detail grids with stateful selection, as opposed to the phone apps which more often navigate on tap to a new detail screen.

It's that latter case that was motivating the overall feature in my mind. (If you have a grid with selection you don't need the extra highlight, as you continue to see the selected row w/its natural selection highlight.)

Reply:
Ah, makes sense! Will adjust